packages/rest/lib/http.cadl

```diff
@@ -80,3 +82,14 @@ model ConflictResponse {
   @header
   statusCode: 409;
 }
+
+// Produces a new model with the same properties as T, but with @query,
+// @header, @body, and @path decorators removed from all properties.
+//
+// ISSUE: Can't use @doc to document this as it leaks into OpenAPI output.
```

Comment:
I agree it would be nice to be able to have a good way to have "library" level comment on how to use things that could be extracted by the language server as well.

Reply:
Filed #131